hyper/http11/connection.py

```diff
@@ -8,15 +8,18 @@
 import logging
 import os
 import socket
+import base64
 
 from .response import HTTP11Response
 from ..tls import wrap_socket
 from ..common.bufsocket import BufferedSocket
-from ..common.exceptions import TLSUpgrade
+from ..common.exceptions import TLSUpgrade, HTTPUpgrade
 from ..common.headers import HTTPHeaderMap
 from ..common.util import to_bytestring
 from ..compat import bytes
 
+from ..http20.connection import H2C_PROTOCOL
```

Comment:
I am assuming you meant tls not tis.  This is where I looked first, but it seemed like the wrong place; however I can't think of a better one.